src/sentry/models/tagvalue.py

```diff
@@ -44,6 +49,10 @@ class Meta:
 
     __repr__ = sane_repr('project_id', 'key', 'value')
 
+    @classmethod
+    def is_valid_value(self, value):
+        return TAG_VALUE_RE.match(value)
```

Comment:
Why not just do `return '\n' not in value` ? It'd be more efficient than the regular expression for now.